test/support/pages/SettingsPage.js

```diff
@@ -7,7 +7,7 @@ define(function (require) {
   var Promise = require('bluebird');
   var Common = require('./Common');
 
-  var defaultTimeout = 60000;
+  var defaultTimeout = 90000;
```

Comment:
I think defaultTimeout should be used when we're looking for an object on the page.  This shouldn't ever need to be more than about 15 seconds.  And we should have a longer timeout when we need to find an object repeatedly until it updates with the expected value (60 seconds should be plenty for our tests).